composer.json

```diff
@@ -20,7 +20,8 @@
         }
     ],
     "require": {
-        "php": "^7.3 || ^8.0"
+        "php": "^7.2 || ^8.0",
```

Comment:
I don't know how this is not detected as a conflict... anyway, please change "7.2" to "7.3".
Also, you need to change the regexp that replace symfony vendor in github actions (it must exclude "deprecation-contracts" since the contracts don't follow the numbering of other components)

Reply:
Done in https://github.com/KnpLabs/KnpMenu/pull/351/commits/5ae265b7ad9105ece977c6a32c0439c4285e49d1

Regarding to github actions, not sure to understand what I need to change, will be in this line?

https://github.com/KnpLabs/KnpMenu/blob/master/.github/workflows/build.yaml#L65

Reply:
exactly that line

Reply:
Not sure how to achieve the exclusion with sed. I try 2 ways based in https://stackoverflow.com/questions/6684857/how-to-skip-lines-matching-a-string and https://unix.stackexchange.com/questions/128010/how-to-sed-e-s-everything-except-a-specific-pattern:

https://github.com/KnpLabs/KnpMenu/pull/351/commits/10569aa7cf379d4d28e86d96a75dd1a0abb04d28

https://github.com/KnpLabs/KnpMenu/pull/351/commits/abdeda84032b13dd5ccefdb562e1ad0b6a524d03

Could you suggest me a way to achieve it?

Reply:
![image](https://user-images.githubusercontent.com/14254/131517640-dfaa1728-8e2e-437b-a4f4-80eea10de714.png)

Reply:
Try with this (but please try it on your local machine, and commit only when it works):

`sed -ri '/symfony\/deprecation-contracts/! s/"symfony\/(.+)": "(.+)"/"symfony\/\1": "'4.4'"/' composer.json;`

reference: https://superuser.com/questions/244723/exclude-some-matches-with-sed